applications/web-browser/page.js

```
@@ -0,0 +1,223 @@
+					"]*" +
+					(end ? "?)(" + end + ")" : ")()"),
+				flags
+			);
+			var handleAttr = function (match, g1, g2) {
+				return (
+					g1 +
+					g2.replace(at1, by2).replace(at2, by2).replace(at3, by2)
+				);
+			};
+		} else {
+			var handleAttr = function (match, g1, g2) {
```

Biome

[error] Shouldn't redeclare 'handleAttr'. Consider to delete it or rename it.

'handleAttr' is defined here:


(lint/suspicious/noRedeclare)